### PR TITLE
Live indicator svg only as tall as required

### PR DESCRIPTION
**Pull request by evoactivity**
The live svg was causing the live button to be ever so slightly taller than required and exposed #689

### CODE REVIEW

File: src/js/media-live-button.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const { MEDIA_TIME_IS_LIVE, MEDIA_PAUSED } = MediaUIAttributes;
 const { MEDIA_SEEK_TO_LIVE_REQUEST, MEDIA_PLAY_REQUEST } = MediaUIEvents;
 
 const indicatorSVG =
-  '<svg viewBox="0 0 6 12"><circle cx="3" cy="6" r="2"></circle></svg>';
+  '<svg viewBox="0 0 4 4" style="width:6px; height:6px;"><circle cx="2" cy="2" r="2"></circle></svg>';
```
Comment (luwes) [inline src/js/media-live-button.js:11]: can this be done without the style attribute? 
might as well change everything in SVG

**Comment by luwes** [comment]
@evoactivity do you have an example online?

I'm not seeing that the media-live-button is making the controlbar higher than it should be
https://github.com/muxinc/media-chrome/pull/700

**Comment by evoactivity** [comment]
@luwes I found what is triggering this for me.

I have this css so the button only shows when on a live stream

```css
media-live-button {
  display: none;
}

media-controller[mediastreamtype='live'] media-live-button {
  display: block;
}
```

If I use `display: flex;` or `dispay: inline-flex;` to show it, the issue is not apparrent, so that is what I'll do for now.

Right now the spacing between the indicator and text is done with an `&nbsp;` spacer slot (and a little padding in the svg). My preference would be to add another css variable and use `gap: var(--media-live-button-gap, 6px);` instead of relying on a non breaking space.

**Comment by luwes** [comment]
the main thing we wanted to prevent was have the live label wrap to the next line I believe.

it's possible with CSS but it would add more code. `&nbsp;` seemed a pretty simple and flexible solution.
you could also swap it out with a normal space if you like it to be wrapped in some cases.

**Comment by evoactivity** [comment]
The element is a flex container though, so wrapping would only be an issue if it was set to block, and setting it to block right now causes the height issue.

I'll draft something up and open it up for discussion.

**Comment by luwes** [comment]
thanks @evoactivity! yes, that's a good idea to move it to a discussion.
closing this PR.

**Pull request closed without merging**